modules/migration-known-issues.adoc

```diff
@@ -31,3 +31,7 @@ ifdef::migrating-3-4[]
 
 * If you cannot install CAM 1.2 on an {product-title} 3 cluster, download the current `operator.yml` file, which fixes this problem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843059[*BZ#1843059*])
 endif::[]
+
+ifdef::migrating-4-2-4[]
+* In the current release (CAM 1.2), you cannot migrate a namespace that contains imagestreams from OpenShift Container Platform 4.4 or later. The following error message is displayed in the Velero Pod log: `Error restoring nametags`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1848561[*BZ#1848561*])
```

Comment:
nit - we usually use {product-title} in place of OpenShift Container Platform. You can leave the version number.

Reply:
Fixed.